### PR TITLE
[WIP] Added number of items found

### PR DESCRIPTION
**Pull request by CloudFenrir**
I added the a label showing the number of items found. I will continue working on the selected items in another commit.

- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for bigger UI changes)

Example image (look at the bottom):
image

### CODE REVIEW

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java [hunk 1]
```diff
@@ -125,7 +124,8 @@ private void initialize() {
                     */
                     if (entriesListView.getItems().size() == 1) {
                         selectAllNewEntries();
-                      }
+                    }
+                    totalItems.textProperty().setValue(String.valueOf(entriesListView.getItems().size()));
```
Comment (tobiasdiez) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java]: It should be possible to bind the text directly to the size:
`totalItems.textProperty().bind(Bindings.size(viewModel.getEntries()))`, which can then be moved to line 90.

Comment (CloudFenrir) [reply src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java]: It actually didn't work as you said, I just pushed some changes that were missing, I hope everything is ok.

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java [hunk 2]
```diff
@@ -103,6 +103,19 @@ private void initialize() {
                     });
                     addToggle.getStyleClass().add("addEntryButton");
                     addToggle.selectedProperty().bindBidirectional(entriesListView.getItemBooleanProperty(entry));
+                    addToggle.selectedProperty().addListener((observable, oldValue, newValue) -> {
+                        if(observable.getValue()){
```
Comment (koppor) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java]: Please follow our code formatting guidelines. The auto formatter should add some spaces here.

**Comment by koppor** [comment]
Happy new year to everyone!

@CloudFenrir May I ask whether you could find some time to finish this contribution?

**Comment by calixtus** [comment]
Sadly, we did not hear from @CloudFenrir anymore. As we like this feature, we decided to adopt this PR.

**Pull request closed without merging**